packages/next/src/build/webpack/loaders/next-metadata-route-loader.ts

```diff
@@ -118,6 +118,46 @@ export async function GET(req, ctx) {
 `
 }
 
+function getDynamicSiteMapRouteCode(resourcePath: string) {
+  // generateSitemaps
+  return `\
```

Comment:
```suggestion
  return `
```
`\` seems wrong here.

Reply:
it's for connecting the new line, I want to start writting code in the new line. this doesn't create `\n`